theseus_gui/src/pages/Settings.vue

```diff
@@ -1,7 +1,286 @@
+      </div>
+      <hr class="card-divider" />
+      <div class="settings-group">
+        <div class="sliders">
+          <span class="slider">
+            Minimum Memory
+            <Slider v-model="javaMemory" :min="1024" :max="8192" :step="1024" />
+          </span>
+          <span class="slider">
+            Maximum Memory
+            <Slider v-model="javaMemory" :min="1024" :max="8192" :step="1024" />
+          </span>
+        </div>
+      </div>
+    </Card>
+    <Card class="settings-card">
+      <h2 class="settings-title">Window</h2>
+      <div class="settings-group">
+        <div class="settings-group">
+          <div class="sliders">
+            <span class="slider">
+              Width
+              <Slider v-model="javaMemory" :min="1024" :max="8192" :step="1024" />
+            </span>
+            <span class="slider">
+              Height
+              <Slider v-model="javaMemory" :min="1024" :max="8192" :step="1024" />
+            </span>
+          </div>
+          <div class="toggle-setting">
+            Start in Fullscreen
+            <input
+              id="fullscreen"
+              v-model="fullscreen"
+              type="checkbox"
+              name="fullscreen"
+              class="switch stylized-toggle"
+            />
+          </div>
+        </div>
+        <hr class="card-divider" />
+        <div class="settings-group">
+          <h3>Console</h3>
+          <div class="toggle-setting">
+            Show console while game is running
+            <input
+              id="fullscreen"
+              v-model="fullscreen"
+              type="checkbox"
+              name="fullscreen"
+              class="switch stylized-toggle"
+            />
+          </div>
+          <div class="toggle-setting">
+            Close console when game quits
+            <input
+              id="fullscreen"
+              v-model="fullscreen"
+              type="checkbox"
+              name="fullscreen"
+              class="switch stylized-toggle"
+            />
+          </div>
+          <div class="toggle-setting">
+            Show console when game crashes
+            <input
+              id="fullscreen"
+              v-model="fullscreen"
+              type="checkbox"
+              name="fullscreen"
+              class="switch stylized-toggle"
+            />
+          </div>
+        </div>
+      </div>
+    </Card>
+    <Card class="settings-card">
+      <h2 class="settings-title">Commands</h2>
+      <div class="settings-group">
+        <div class="toggle-setting">
+          Pre Launch
+          <input v-model="javaArgs" type="text" class="input" />
+        </div>
+        <div class="toggle-setting">
+          Wrapper
+          <input v-model="javaArgs" type="text" class="input" />
+        </div>
+        <div class="toggle-setting">
+          Post Launch
+          <input v-model="javaArgs" type="text" class="input" />
+        </div>
```

Comment:
These all v-model the same thing, so all the checkboxes move when you click one

Reply:
I'm waiting for settings to be exposed by Tauri before worrying too much about the data behind the page. Now that it's exposed in settings.js and the Win10 bug is fixed, I'll wire it up.